.gitignore

```diff
@@ -94,3 +94,7 @@ chia-blockchain-gui/src/locales/_build
 build_scripts\win_build
 build_scripts/win_build
 win_code_sign_cert.p12
+
+# Certificates
+mozilla-ca
```

Comment:
Should we really gitignore this? I don't actually remember what I did with -gui..